### DISCUSSION TITLE
Hi, is opencart still maintained by the owner?

### DISCUSSION THREAD

**Discussion by newbie7777** [opencart/opencart#12185] [General]
Hi, bit of a dumb question, but isn't opencart still maintained by the old developers and not by someone else? A few days ago I was talking with colleagues and they shared that Opencart is maintained by other people and another company and the quality will most likely go down. I hope this is all a joke. Thanks to Opencart it has been giving me bread and food for over 10 years now.

**Comment by felipoantonoff** [comment]
Hello, as far as I know Opencart is still maintained by its creator Daniel, his team and other members of the community.
See the Commits most of them are from Daniel and other members.

Even its latest version has support for PHP 8 and with several current technologies, in addition to having a huge ecosystem of plugins / apps and professionals, so the future of Opencart we can say that it is very good, in addition to being a mature E-commerce system and consolidated, probably the best E-commerce system with a focus on performance, ease of use and modularity.

**Comment by IP-CAM** [comment]
Well, Daniel is still active, in contrary to the vast majority of former famous Opencart contributors. And for about two years now, he's trying to get Opencart v.4.x working, so far with rather mixed results. This, most likely due to the very obvious reason, that many former helpers no longer seem to be willing, to participate on his Project. But that's only my personal impression, after being around OC for a while already...
